src/plugins/Workers/SyncWorker.js

```diff
@@ -101,42 +111,73 @@ class SyncWorker extends Worker {
 
   async execAddressFetching() {
     const self = this;
-    const { addresses } = this.storage.getStore().wallets[this.walletId];
+    const { addresses, network } = this.storage.getStore().wallets[this.walletId];
+    const currBlockheight = this.storage.getStore().chains[network.toString()].blockheight;
+    const currTime = Date.now();
     const { fetchAddressInfo } = this;
 
     const toFetchAddresses = [];
 
-    let prevWasUsed = false;
-    Object.keys(addresses).forEach((walletType) => {
+    let nbPreviousUsed = 0;
```

Comment:
What's nb?

Reply:
numberOfPreviousUsed, I actually just checked that out, the official abbreviation for number seems to be no, I guess I just realized I'm puttin some 'nb' quite a lot since ages thinking that it's good abbreviation, it is not :x

Reply:
Both of them are not great abbreviations

Reply:
Would love to fix that, what do you suggest, I'm counting the number of previous (it's a loop) used address.